pkg/jx/cmd/common_install.go

```diff
@@ -505,10 +505,15 @@ func (o *CommonOptions) installOc() error {
 // get the latest version from kubernetes, parse it and return it
 func (o *CommonOptions) getLatestVersionFromKubernetesReleaseUrl() (sem semver.Version, err error) {
 	response, err := http.Get(stableKubeCtlVersionURL)
```

Comment:
Potentially we can tests this with `httptest` and simulate when the download address is down. It will most likely block for a while, but the TCP timeout on OS level will kick in and it will cut the connection at some point.

Reply:
I think that is worth a followup ticket and change, this is currently getting large-ish as is.